include/deal.II/base/bounding_box.h

```diff
@@ -170,10 +169,9 @@ inline BoundingBox<spacedim, Number>::BoundingBox(
 {
   // We check the Bounding Box is not degenerate
   for (unsigned int i = 0; i < spacedim; ++i)
-    Assert(
-      boundary_points.first[i] <= boundary_points.second[i],
-      ExcMessage(
-        "Bounding Box can't be created: the point's order should be bottom left, top right!"));
+    Assert(boundary_points.first[i] <= boundary_points.second[i],
+           ExcMessage("Bounding Box can't be created: the point's "
+                      "order should be bottom left, top right!"));
```

Comment:
This I fixed additionally manually.

Reply:
A nit-pick: Shouldn't this be "the points' order"?

Reply:
> A nit-pick: Shouldn't this be "the points' order"?

Sure. :smiley:

Reply:
I've taken to just editing people's patches for these obvious things. The github editor is good enough for that.